pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -1,97 +1,109 @@
-#!/usr/bin/env nix-shell
-#!nix-shell update-shell.nix -i python3
-
+#!/usr/bin/env python
 
+# run with:
+# $ nix run .\#vimPluginsUpdater
 # format:
-# $ nix run nixpkgs.python3Packages.black -c black update.py
+# $ nix run nixpkgs#python3Packages.black -- update.py
 # type-check:
-# $ nix run nixpkgs.python3Packages.mypy -c mypy update.py
+# $ nix run nixpkgs#python3Packages.mypy -- update.py
 # linted:
-# $ nix run nixpkgs.python3Packages.flake8 -c flake8 --ignore E501,E265,E402 update.py
+# $ nix run nixpkgs#python3Packages.flake8 -- --ignore E501,E265,E402 update.py
 
-# If you see `HTTP Error 429: too many requests` errors while running this script,
-# refer to:
+# If you see `HTTP Error 429: too many requests` errors while running this
+# script, refer to:
 #
 # https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/vim.section.md#updating-plugins-in-nixpkgs-updating-plugins-in-nixpkgs
 #
-# (or the equivalent file /doc/languages-frameworks/vim.section.md from Nixpkgs master tree).
+# (or the equivalent file /doc/languages-frameworks/vim.section.md
+# from Nixpkgs master tree).
 #
 
 import inspect
 import os
-import sys
 import logging
-import subprocess
 import textwrap
 from typing import List, Tuple
 from pathlib import Path
 
-import git
 
 log = logging.getLogger()
 
 sh = logging.StreamHandler()
-formatter = logging.Formatter('%(name)s:%(levelname)s: %(message)s')
+formatter = logging.Formatter("%(name)s:%(levelname)s: %(message)s")
 sh.setFormatter(formatter)
 log.addHandler(sh)
 
 # Import plugin update library from maintainers/scripts/pluginupdate.py
 ROOT = Path(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))))
-# Ideally, ROOT.(parent^5) points to root of Nixpkgs official tree
-sys.path.insert(0, os.path.join(ROOT.parent.parent.parent.parent.parent, "maintainers", "scripts"))
 import pluginupdate
+import importlib
 from pluginupdate import run_nix_expr, PluginDesc
+nvim_treesitter = importlib.import_module("treesitter")
```

Comment:
is this different from just `import`?

Reply:
at first I was importing `nvim-treesitter` and python doesnt accept imports with dashes hence the importlib call. After that I just imported "treesitter" and didnt revert the change. Will fix too.